src/box/lua/cfg.cc

```diff
@@ -81,6 +81,14 @@ lbox_cfg_set_bootstrap_strategy(struct lua_State *L)
 	return 0;
 }
 
+static int
+lbox_cfg_set_bootstrap_leader(struct lua_State *L)
+{
+	if (box_set_bootstrap_leader() != 0)
```

Comment:
Shouldn't we warn, that option is NoOp when bootstrap is already done? There's no mention about this in documentation, and smb may think, that changing bootstrap_leader is a way of rebootstrap.

What I'm trying to say, it's non obvious on the first glance for a user, why the option is dynamic

Reply:
Good point, I've added a small docbot request to the commit message.